designs/virtualization/4.7/create-vm-from-template/create-virtual-machine-from-template.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: Virtualization
+version: 4.7
+---
+# Add an operating system image to a common template
```

Comment:
@fkargbo I don't think this is the right markdown content, is it? it seems like this is about image upload.

Reply:
I'm not sure what happened here. I'll check the markdown file. thanks.